bootstrap/src/lib/cache-warmer/side-effects.ts

```diff
@@ -21,7 +21,7 @@ import {
   warmupUnsubscribed,
 } from './actions'
 import { WARMUP_REQUEST_ID, Config, get } from './config'
-import { RootState } from './reducer'
+import { RootState } from '../../index'
```

Comment:
Please revert changes to this file.

We need to be aware of what depends on what. This warmer component needs to be contained, it should not depend on the application, but the application depends on the warmer component.

Reply:
Yep, I know, but this was breaking the cache warmer. We need to tell somehow to `createEpicMiddleware` that Root State now has rate-limit as well, as both stores and reducers are combined.